week2/homework/src/index.js

```diff
@@ -1,3 +1,118 @@
 'use strict';
+let fs = require('fs');
+let order = process.argv[2];
+let orderDefinition = process.argv[3];
+let orderDefinition2 = process.argv[4];
 
-// TODO: Write the homework code in this file
+fs.readFile('./toDo.json', 'utf8', function (error, data) {
+  if (error) {
+    console.log(error);
+  }
+  else {
```

Comment:
This code can make use of [early return](https://dev.to/jenniferlynparsons/early-returns-in-javascript-5hfb) to make clear what you're doing.
Nesting everything inside an `else` makes the code harder to read. I use early returns all the time!

Reply:
I have tried to change the program a bit but still some code are nested to other. Is it okey or should i need to change more?

Reply:
can you git push your changes so i can see them? :)